Use of internal package.

The copy of robustio/renameio packages imports internal API, which is missing in this module.

```
github.com/goproxyio/goproxy/v2@v2.0.1/robustio/robustio_windows.go:9:2: use of internal package internal/syscall/windows not allowed
```

https://github.com/goproxyio/goproxy/blob/9a7bb4c0c90f2c4a2a5c8855ad4cea40397f0107/robustio/robustio_windows.go#L7-L15

--- a/go.mod
+++ b/go.mod
@@ -2,4 +2,7 @@ module github.com/goproxyio/goproxy
 
 go 1.12
 
-require golang.org/x/mod v0.1.0
+require (
+	github.com/goproxyio/windows v0.0.0-20191126033816-f4a809841617
+	golang.org/x/mod v0.1.0
+)

--- a/robustio/robustio_windows.go
+++ b/robustio/robustio_windows.go
@@ -6,12 +6,13 @@ package robustio
 
 import (
 	"errors"
-	"internal/syscall/windows"
 	"io/ioutil"
 	"math/rand"
 	"os"
 	"syscall"
 	"time"
+
+	"github.com/goproxyio/windows"
 )
 
 const arbitraryTimeout = 500 * time.Millisecond

